src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/grid/mass-actions.js

```diff
@@ -96,6 +102,12 @@ define(['jquery', 'underscore', 'oro/translator', 'backbone', 'pim/form', 'pim/t
       this.count = this.collection.state.totalRecords;
       this.collection.trigger('backgrid:selectAll');
 
+      switch (this.collection.inputName) {
+        case 'product-grid':
+          analytics.track('product-grid:product:all-selected');
+          break;
+      }
+
```

Comment:
could we track a more generic event like below:
```suggestion
          analytics.track('grid:item:all-selected', {
             inputName: this.collection.inputName,
          });

```

We should handle the product-grid case in the "implementation" side